wadium/story/models.py

```diff
@@ -1,36 +1,18 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.utils import timezone
-import secrets
-
-def generate_uid():
-    return secrets.token_hex(6)
 
 class Story(models.Model):
-    uid  = models.CharField(max_length=100, unique=True, default=generate_uid)
     writer = models.ForeignKey(User, related_name='stories', on_delete=models.CASCADE)
     title = models.CharField(max_length=100, db_index=True)
     subtitle = models.CharField(max_length=140, blank=True)
-    featured_image = models.PositiveSmallIntegerField(null=True) # put the order of the block that will be the feature image
+    body = models.JSONField(default={})
```

Comment:
There's 
WARNINGS:
story.Story.body: (fields.E010) JSONField default should be a callable instead of an instance so that it's not shared between all field instances.   
        HINT: Use a callable instead, e.g., use `dict` instead of `{}`.
Warning because of `default`.